tests/acceptance/bootstrap/AuthAppContext.php

```diff
@@ -50,6 +50,7 @@ public function before(BeforeScenarioScope $scope): void {
 
 	/**
 	 * @When user :user creates app token with expiration time :expiration using the auth-app API
+	 * @When user :user tries to create app token with expiration time :expiration using the auth-app API
```

Comment:
```suggestion
* @When /^user "([^"]*)" (?:creates|tries to create) app token with expiration time "([^"]*)" using the auth-app API$/

```
Will this work for both?

Reply:
hmm, I prefer having both on separate lines because it's more readable than having regex.
But it's up to you.